sentinelhub/api/batch_base.py

```diff
@@ -0,0 +1,98 @@
+"""
```

Comment:
I'm wondering if module name `base_batch.py` would be better. This way it would be more in-line with `base.py` and `base_request.py`.

Reply:
Moving things into a subfolder did the job :+1: